.changeset/kind-eggs-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/components": patch
+---
+
+Fix toast notification
```

Comment:
Since this description goes into the changelog, we would prefer it to have some extra details (eg. what are you fixing?) :)